uniparc-rest/src/main/java/org/uniprot/api/uniparc/request/UniParcGetByDBRefIdRequest.java

```diff
@@ -0,0 +1,29 @@
+package org.uniprot.api.uniparc.request;
+
+import javax.validation.constraints.NotNull;
+
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import io.swagger.v3.oas.annotations.Parameter;
+
+/**
+ * @author sahmad
+ * @created 13/08/2020
+ */
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class UniParcGetByDBRefIdRequest extends UniParcGetByIdPageSearchRequest {
+    @Parameter(hidden = true)
+    private static final String DB_ID_STR = "dbid";
+
+    @Parameter(
+            description =
+                    "All UniParc cross reference accessions, eg. AAC02967 (EMBL) or XP_006524055 (RefSeq)")
```

Comment:
not sure if we should call them accessions? maybe it is fine, but perhaps we should replace "accessions" with "id" in the description?